### ISSUE TITLE
selector-type-no-unknown doesn't recognize Sass placeholder selector

### ISSUE THREAD

**Issue by efedorenko** [stylelint/stylelint#1334]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

`selector-type-no-unknown` rule doesn't recognize [Sass placeholder selector](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_) (`%foo`).

> Which rule, if any, is this issue related to?

`selector-type-no-unknown`

> What CSS is needed to reproduce this issue?

```sass
%icon-loading {
  ...
}
```

> Which version of stylelint are you using?

`6.4.2`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

NPM script with `stylelint --syntax scss 'app/assets/stylesheets/scss/**/*.scss'`.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to Sass syntax.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warning was flagged:

```shell
738:1   ⚠  Unexpected unknown type selector "%icon-loading"   selector-type-no-unknown
```
